src/screens/simulation/PublicSimulation.js

```diff
@@ -433,10 +434,23 @@ const getTimeframesFromValues = ({
         start_date: deconfinement_start_date,
         name: 'Déconfinement',
         lim_time: 365 + differenceInDays(new Date(), initial_start_date),
-        enabled: lockdown_enabled && deconfinement_enabled,
+        enabled: deconfinement_enabled,
     },
 ];
 
+const decorator = createDecorator({
+    field: 'lockdown_enabled',
+    updates: {
+        deconfinement_enabled: (lockdown_enabled, { deconfinement_enabled }) => {
+            if (!lockdown_enabled && deconfinement_enabled) {
```

Comment:
This is a bit too complex for me
Why not just return lockdown && deconfinement ?